src/etl/rds.py

```diff
@@ -103,11 +105,14 @@ def persist_data(self, datum):
         self.validate_json("Parameters", datum.parameters)
         self.validate_json("JSON Data", datum.content)
 
+        # ensure that uuid is in uuid4 format
+        self.validate_uuid(datum.uuid)
+
         insert_json_data = """
             INSERT INTO capture.json_data 
             (start_time, response_time, response_code, url, api,
              script_name, script_pid,
-             parameters, json_content)
+             parameters, json_content, uuid)
             VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s)
```

Comment:
With `uuid` added, another `%s` should be added as well.

Reply:
Fixed and working on test now ...